packages/kbld-config.yaml

```diff
@@ -23,7 +23,7 @@ overrides:
     newImage: ""
   - image: tanzu-cli-plugins/kubernetes-release-windows-amd64:latest
     newImage: ""
-  #! addons-manager
+  #! addons-manager-v2
   - image: addons-controller:latest
```

Comment:
Should we also update this image? It also need to be updated in https://github.com/vmware-tanzu/tanzu-framework/blob/8cc0ebf22350dbddb3e85f5330ce95437e38bc30/packages/management/addons-manager-v2/bundle/config/upstream/addons-manager.yaml#L173

Reply:
seems like it is the same

Reply:
Not sure what the build process of management-package is. Might need to confirm how they substitute these images.

Reply:
I mean should we add a `v2` to the image

Reply:
@blc1996 @vijaykatam I think they are using the built image from addons folder which is this line:
https://github.com/vmware-tanzu/tanzu-framework/blob/main/Makefile#L673
correct me if I'm wrong

Reply:
I think same image name is fine.

Reply:
Might want to update the git comment. As an outsider just droping inn to review, it seems to me that the test results  would changer from

```NAME                                                DISPLAY-NAME                       SHORT-DESCRIPTION                                                                                                            LATEST-VERSION                NAMESPACE
addons-manager.tanzu.vmware.com                     tanzu-addons-manager               This package provides TKG addons lifecycle management capabilities.    
```
to

```NAME                                                DISPLAY-NAME                       SHORT-DESCRIPTION                                                                                                            LATEST-VERSION                NAMESPACE
addons-manager.tanzu.vmware.com                     tanzu-addons-manager-v2               This package provides TKG addons lifecycle management capabilities.    
```

Reply:
@adduarte the `display-name` here is from metadata.yaml (link below) and I think it's ok to keep it same since it's under management package repo

https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/addons-manager/metadata.yaml#L7